test/ui/cypress/support/auth.ts

```diff
@@ -0,0 +1,63 @@
+Cypress.Commands.add('requiredVariablesAreSet', (requiredVariables: string[]) => {
+    requiredVariables.forEach((variable) => {
+      if (!Cypress.env(variable)) {
+        throw new Error(`Missing required environment variable: ${variable}`);
+      }
+    });
+});
+
+Cypress.Commands.add('loginWithRHAccount', () => {
+    cy.requiredVariablesAreSet([
+        'baseUrl',
```

Comment:
Why this inconsistent naming? Also, the project already defined and uses env variable `DEPLOYMENT_DRIVER_URL` so we should use that name instead.